iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -60,12 +61,12 @@ void publisher()
     iox::popo::Publisher<TransmissionData_t> publisher({"Single", "Process", "Demo"}, publisherOptions);
 
     uint64_t counter{0};
-    std::string greenRightArrow("\033[32m->\033[m ");
+    constexpr const char GREEN_RIGHT_ARROW[] = "\033[32m->\033[m ";
     while (keepRunning.load())
     {
         publisher.loan().and_then([&](auto& sample) {
             sample->counter = counter++;
-            consoleOutput(std::string("Sending   " + greenRightArrow + std::to_string(sample->counter)));
+            consoleOutput("Sending ", GREEN_RIGHT_ARROW, sample->counter);
```

Comment:
please add tow additional spaces to `Sending ` to have the arrows aligned and make the test work again